styles.css

```diff
@@ -0,0 +1,138 @@
+body {
+  height: 100%;
+  max-width: 100%;
+  font-family: "Roboto", sans-serif;
+}
+nav {
+  top: 0px;
+  width: 100%;
+  display: flex;
+}
+#youtube-logo {
```

Comment:
Like my colleague told above, you should avoid using id's because they need to be unique, and you might have to re-use these CSS styling on other elements, especially if the project is bigger than this one. Use classes instead.